### PR TITLE
first task

### PR DESCRIPTION
**Pull request by Unshack**
work on the main page

### CODE REVIEW

File: frontend/src/data/products.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ export const services = [
   },
   {
     id: 5,
-    title: "secure payments",
+    title: "secure payments & Services",
```
Comment (DenisWr) [inline frontend/src/data/products.js:252]: consistency

File: frontend/src/data/products.js [hunk 2]
```diff
@@ -232,7 +232,7 @@ export const services = [
   {
     id: 2,
     title: "daily surprise offers",
-    description: "save up to 25%",
+    description: "save up to 35%",
```
Comment (DenisWr) [inline frontend/src/data/products.js:235]: bring back it to 25%

**Review by DenisWr — APPROVED** [review]
Great work, we need more upgraded UI design

File: frontend/src/data/products.js (hunk 1 above)
Comment (Unshack) [reply frontend/src/data/products.js:252]: assign to lamlam

**Pull request merged**